pkg/bootstrap/api/helpers.go

```diff
@@ -32,3 +33,13 @@ func ValidateBootstrapGroupName(name string) error {
 	}
 	return fmt.Errorf("bootstrap group %q is invalid (must match %s)", name, BootstrapGroupPattern)
 }
+
+// IsValidateUsage validate if the usage is valid
+func IsValidateUsage(usage string) bool {
```

Comment:
This func should be named `ValidateUsages` and consume a _slice of strings_ with the logic you have above.

Reply:
I suggested to use `IsValidateUsage` (while I was thinking of `IsValidUsage`) if it consumes a single usage. I'm +1 for using `ValidateUsages` and consuming usages as a slice of strings.